pkg/providers/pricing/zz_generated.pricing.go

```diff
@@ -18,9 +18,9 @@ package pricing
 
 import "time"
 
-// generated at 2023-08-21T14:24:21Z for us-east-1
+// generated at 2023-09-19T22:00:25Z for us-east-1
 
-var initialPriceUpdate, _ = time.Parse(time.RFC3339, "2023-08-21T14:24:21Z")
+var initialPriceUpdate, _ = time.Parse(time.RFC3339, "2023-09-19T22:00:25Z")
```

Comment:
Same comment as the bandwidth gen. I think we can drop the auto-gened changes as part of this PR.